src/mpiwrap/message_passing.F

```diff
@@ -59,55 +57,15 @@ MODULE message_passing
 #endif
 
 #if defined(__parallel)
+! subroutines: unfortunately, mpi implementations do not provide interfaces for all subroutines (problems with types and ranks explosion),
+!              we do not quite know what is in the module, so we can not include any....
+!              to nevertheless get checking for what is included, we use the mpi module without use clause, getting all there is
```

Comment:
Is there a combination of compiler+MPI implementation where this is not possible with the `mpi_f08` module?

Reply:
Thanks for asking!

This PR is just a draft yet, but it seems messy for instance OpenMPI 5.x misses `mpi_get_address` in the F08 interface (though, it does not harm the `USE` statement requesting it). The `MPI_STATUSES_IGNORE` situation is even more crowded, i.e., using an own variable with some kind of content denoting "ignore" seems to be unportable in general beside of `MPI_STATUSES_IGNORE` missing even from the pre-F08 interface.

I will play a bit more with this PR and worst case closing it without merge. What I am testing locally (beside of CI here), is ARM64 w/ OpenMPI from Ubuntu distro, Intel MPI and OpenMPI on x86. I extend this to more specific tests with OMPI4 and stock-MPICH.

Reply:
Let's see when or whether this part of the MPI standard is available entirely (apart from hitting compiler bugs as we had them with MPICH+GCC 11 (or was it GCC 12?)).